src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -71,20 +73,21 @@ const ChatBtn = styled.img`
 const HeaderTitle = styled.div`
   display: flex;
   text-align: left;
-  margin: 0 auto;
-  font-weight: 900;
-  line-height: 17px;
+  justify-content: center ;
+  font-weight: 700;
+  line-height: 57px;
   font-family: 'Poppins', sans-serif;
 `
 const HeaderSubTitle = styled.div`
   display: flex;
   text-align: left;
+  justify-content: center ;
   font-size: 13px;
   font-weight: 400;
   color: var(--headertext);
-  margin-left: 1px;
   margin-top: 3px;
-  line-height: 19px;
+  padding-bottom: 3px;
+  line-height: 45px;
```

Comment:
Please change `line-height` value for default `line-height` values, you can use `line-height: 3.46` instead of use px

![imagen](https://user-images.githubusercontent.com/66505715/160884552-9e6ad019-b56c-4ffe-bc3c-91ea2375aea6.png)